packages/@expo/metro-config/src/transform-plugins/__tests__/__snapshots__/import-export-plugin-babel-snapshots.test.ts.snap

```diff
@@ -0,0 +1,2029 @@
+  var n = {};
+  if (e) Object.keys(e).forEach(function (k) {
+    n[k] = e[k];
+  });
+  n.default = e;
+  return n;
+}
+Object.defineProperty(exports, "foo", {
+  enumerable: true,
+  get: function () {
+    return foo;
+  }
+});
+var _moduleWithGetter = require("./moduleWithGetter");
+var foo = _interopNamespace(_moduleWithGetter);"
+`;
+
+exports[`misc export-let-function-name 1`] = `
+"Object.defineProperty(exports, '__esModule', {
+  value: true
+});
+Object.defineProperty(exports, "a", {
+  enumerable: true,
+  get: function () {
+    return a;
+  }
+});
+Object.defineProperty(exports, "b", {
+  enumerable: true,
+  get: function () {
+    return b;
+  }
+});
+Object.defineProperty(exports, "c", {
+  enumerable: true,
+  get: function () {
+    return c;
+  }
+});
+Object.defineProperty(exports, "d", {
+  enumerable: true,
+  get: function () {
+    return d;
+  }
+});
+Object.defineProperty(exports, "e", {
+  enumerable: true,
+  get: function () {
+    return e;
+  }
+});
+let a = function () {};
+let b = function X() {};
+let c = () => {};
+let d = class {};
+let e = class Y {};"
+`;
+
+exports[`misc export-let-function-name-with-fn-name-transform 1`] = `
+"Object.defineProperty(exports, '__esModule', {
+  value: true
+});
+Object.defineProperty(exports, "a", {
+  enumerable: true,
+  get: function () {
+    return a;
+  }
+});
+Object.defineProperty(exports, "b", {
+  enumerable: true,
+  get: function () {
+    return b;
+  }
+});
+Object.defineProperty(exports, "c", {
+  enumerable: true,
+  get: function () {
+    return c;
+  }
+});
+Object.defineProperty(exports, "d", {
+  enumerable: true,
+  get: function () {
+    return d;
+  }
+});
+Object.defineProperty(exports, "e", {
+  enumerable: true,
+  get: function () {
+    return e;
+  }
+});
+let a = function () {};
+let b = function X() {};
+let c = () => {};
+let d = class {};
+let e = class Y {};"
+`;
+
+exports[`misc export-var-and-modify 1`] = `
+"Object.defineProperty(exports, '__esModule', {
+  value: true
+});
+Object.defineProperty(exports, "a", {
+  enumerable: true,
+  get: function () {
+    return a;
+  }
+});
+var a;
+a += 1;
+a -= 1;
+a *= 1;
+a /= 1;
+a %= 1;
+a **= 1;
+a <<= 1;
+a >>= 1;
+a >>>= 1;
+a &= 1;
+a ^= 1;
+a |= 1;
+a &&= 1;
+a ||= 1;
+a ??= 1;"
+`;
+
+exports[`misc for-of-in-export 1`] = `
+"Object.defineProperty(exports, '__esModule', {
+  value: true
+});
+Object.defineProperty(exports, "foo", {
+  enumerable: true,
+  get: function () {
+    return foo;
+  }
+});
+Object.defineProperty(exports, "bar", {
+  enumerable: true,
+  get: function () {
+    return foo;
+  }
+});
+let foo;
+for (foo of []) {}
+for (foo in []) {}
+for (foo of []) {
+  let foo;
+}
+for ({
+  foo
+} of []) {}
+for ({
+  foo
+} of []) {
+  let foo;
+}
+for ({
+  test: {
+    foo
+  }
+} of []) {}
+for ([foo, [...foo]] of []) {}
+for ([foo, [...foo]] of []) {
+  let foo;
+}
+for (foo of []);
+{
+  let foo;
+  for (foo of []) {}
+}"
+`;
+
+exports[`misc import-const-throw 1`] = `
```

Comment:
Babel adds a runtime error in the case of reassigning the imported value. Rollup throws during bundling. We currently transform this invalid syntax, but the behavior is undefined.